generators/app/templates/.gitignore

```diff
@@ -24,33 +24,26 @@ Thumbs.db
 # ---------------------------
 /craft/plugins/*
 /craft/rev-manifest.json
-/craft/storage/
+/craft/storage/*<% } %>
```

Comment:
I think this `<% } %>` closing tag should actually go after the `Craft Public` section.